src/main/java/web_layer/controllers/UserMvcController.java

```diff
@@ -29,10 +28,6 @@ public ModelAndView getLoginPage() {
     @GetMapping("/")
     public String getHomePage(Principal crtUser) {
         UserDto user = userService.getUserByUsername(crtUser.getName());
-        if (user instanceof StudentDto) {
-            return "redirect:/student/timeline";
-        } else {
-            return "redirect:/professor/dashboard";
-        }
+        return "redirect:/" + user.getRole().toLowerCase() + "/";
```

Comment:
Extract this string in a public static final variable.

Reply:
explain please

Reply:
Create public static final String REDIRECT = "redirect:/" in the Constants class.